source/stylesheets/_navigation.scss

```diff
@@ -0,0 +1,210 @@
+
+// ==== main nav =====
+.nav-title{
+	color: $text-color-lighter;
+	font-weight: $medium-weight;
+	border-bottom: none;
+	padding: $medium-padding;
+}
+
+ul{
+	list-style-type: none;
+    @extend .center-object;
+}
+
+li {
+	float: left;
```

Comment:
Let's discuss what the goal is here. I feel like float left is odd to use in conjunction with the flex properties you get from the above `@extend .center-object` on the `ul`.